egs/swbd_fisher/run.sh

```diff
@@ -54,6 +54,16 @@ for order in 3 4 5; do
 
   make_lm_dir.py --num-splits=${splits} --keep-splits=true data/counts_40k_${order} \
      data/optimize_40k_${order}/final.metaparams data/lm_40k_${order}
+
+  mkdir -p data/arpa
+  format_arpa_lm.py data/lm_40k_${order} | gzip -c > data/arpa/poco_poco_combination.${order}g.gz
+  
+  echo "Perplexity for poco-with-poco-combination ${order}-gram before pruning"
```

Comment:
I think names like poco-with-poco-combination are confusing in this context.  Maybe best to give this a more normal message, but have some comments somewhere else explaining how to gather all the values you need for this table (and maybe put the data of the table itself somewhere in a comment).